deploy/webhook-operator.yaml

```diff
@@ -1202,7 +1202,15 @@ rules:
 - apiGroups:
   - ""
   resources:
-  - services
+  - nodes
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - admissionregistration.k8s.io
```

Comment:
is this a merge conflict resolution? It doesn't seem like we need to modify the webhook-operator yaml to touch the `mutatingwebhookconfigurations` APIs do we?

Reply:
We didn't touch `mutatingwebhookconfigurations` in this PR, it seems to be auto-generated from `make bundle`

Reply:
Ah I know now. Because adding the `node` RBAC rules added a first item to the `ClusterRole` the manager is running as, we need to adjust the webhook operator's kustomize manifests:
```
diff --git a/deploy/overlays/webhook/role.yaml b/deploy/overlays/webhook/role.yaml
index 97221506..f5c1444a 100644
--- a/deploy/overlays/webhook/role.yaml
+++ b/deploy/overlays/webhook/role.yaml
@@ -1,3 +1,3 @@
 ---
-- path: /rules/1
+- path: /rules/2
   op: remove
```
apply the above, then run `make deployments`. That did the trick for me.